addons/pos_sale/static/src/overrides/components/control_buttons/control_buttons.js

```diff
@@ -8,15 +8,18 @@ patch(ControlButtons.prototype, {
     onClickQuotation() {
         this.dialog.add(SelectCreateDialog, {
             resModel: "sale.order",
-            noCreate: true,
             multiSelect: false,
+            noCreate: true,
             domain: [
                 ["state", "!=", "cancel"],
-                ["invoice_status", "!=", "invoiced"],
+                ["amount_unpaid", ">", 0],
             ],
             onSelected: async (resIds) => {
-                await this.pos.onClickSaleOrder(resIds[0]);
+                // a callback is needed to keep the dialog open otherwise it will close due to executeOnceAndClose of SelectCreateDialog
```

Comment:
Instead of this, maybe it's nice to consider introducing a new props in the `SelectCreateDialog` component, say, `keepOpenAfterSelect` and overidding the `select` method such that if `props.keepOpenAfterSelect` is `true`, we only execute the `onSelected` callback but we don't close the dialog.

Reply:
And don't forget the case that when the user selects and option from the SelectionPopup, the `SelectCreateDialog` should close.